src/mon/PGMap.cc

```diff
@@ -1698,6 +1700,8 @@ void PGMap::dump_pg_stats_plain(
           << st.last_deep_scrub
           << st.last_deep_scrub_stamp
           << st.snaptrimq_len
+          << st.scrub_begin_stamp
```

Comment:
For this human-readable output, it might be more helpful to show the duration directly (if end_stamp > begin_stamp). The plain pg dump output is getting pretty wide on a terminal, but scrub duration is meaningful to a user - helps they see e.g. why scrubs may not be completing across all pgs in a couple weeks.

Reply:
Okay got it. I'll make the change.